lib/node/runner/node_runner.go

```diff
@@ -158,6 +159,19 @@ func (nr *NodeRunner) Ready() {
 		return
 	}
 
+	//CORS
+	{
+		allowedOrigins := ghandlers.AllowedOrigins([]string{"*"})
+		allowedMethods := ghandlers.AllowedMethods([]string{"GET", "HEAD", "POST", "PUT", "DELETE", "OPTIONS"})
```

Comment:
We should consider the available methods which are needed at this time. I think we can remove the methods except `GET` and `POST`.

Reply:
Oh Thanks! @spikeekips :->